Cakefile

```diff
@@ -54,6 +54,10 @@ codeFor = ->
     js = js.replace /^\/\/ generated.*?\n/i, ''
 
     cshtml = "<pre><code>#{hljs.highlight('coffeescript', cs).value}</code></pre>"
+    # Temporary fix until highlight.js adds support for newer CoffeeScript reserved words
```

Comment:
They’re actually not the same. We need to search for the space after the keyword, or else `as` matches within `class="`. But if we replace including the space, we get `<span class="reserved">as </span>`. Maybe this could be done with an even cleverer regex, but I’m not sure it’s worth the effort.